consensus/parlia/parlia.go

```diff
@@ -975,8 +1405,11 @@ func CalcDifficulty(snap *Snapshot, signer common.Address) *big.Int {
 }
 
 // SealHash returns the hash of a block prior to it being sealed.
-func (p *Parlia) SealHash(header *types.Header) common.Hash {
-	return SealHash(header, p.chainConfig.ChainID)
+func (p *Parlia) SealHash(header *types.Header) (hash common.Hash) {
+	hasher := sha3.NewLegacyKeccak256()
+	encodeSigHeaderWithoutVoteAttestation(hasher, header, p.chainConfig.ChainID)
```

Comment:
The original `SealHash` still work, doesn't ?

Currently sealHash function does not include the validator and attention, which is not secure.

Reply:
Yes, we have only changed the SealHash func used in the miner module to record which block the miner should seal, the remaining logic still uses the original SealHash func.